context/brand-context/default/scss/10-settings/rfs.scss

```diff
@@ -0,0 +1,69 @@
+/**
+ * Responsive Font Sizes v8.0.4
+ * https://github.com/twbs/rfs
+ * https://github.com/twbs/rfs/blob/main/LICENSE
+ */
+
+// Configuration
+// Adapted from:
+// https://github.com/twbs/rfs/blob/v8.0.4/scss.scss
+
+// Base font size
+$rfs-base-font-size: 1.25rem;
+$rfs-font-size-unit: rem;
+
+@if $rfs-font-size-unit != rem and $rfs-font-size-unit != px {
+  @error "`#{$rfs-font-size-unit}` is not a valid unit for $rfs-font-size-unit. Use `px` or `rem`.";
+}
+
+// Breakpoint at where font-size starts decreasing if screen width is smaller
+$rfs-breakpoint: 1200px;
+$rfs-breakpoint-unit: px;
+
+@if $rfs-breakpoint-unit != px and $rfs-breakpoint-unit != em and $rfs-breakpoint-unit != rem {
+  @error "`#{$rfs-breakpoint-unit}` is not a valid unit for $rfs-breakpoint-unit. Use `px`, `em` or `rem`.";
+}
+
+// Resize font size based on screen height and width
+$rfs-two-dimensional: false;
+
+// Factor of decrease
+$rfs-factor: 10;
```

Comment:
springer currently sets this as `1.6` as I'm guessing `10` was too extreme. Was this the value copied directly from their source code or did you decide upon this as the default?

Reply:
This is in the `default` brand, and is an exact copy of the `rfs` defaults. There is a `springer/scss/10-settings/rfs.scss` file that sets the springer settings to override these

Reply:
oh yeah I just meant maybe we should default to a low factor and let people choose more extreme factors if they have a specific need